internals/get-package-json/package.json

```diff
@@ -8,7 +8,9 @@
     "dist"
   ],
   "scripts": {
-    "build": "tsc",
+    "build": "run-p build:*",
+    "build:js": "node ../../scripts/esbuild.mjs",
+    "build:types": "tsc --declaration --emitDeclarationOnly",
```

Comment:
Should these flags be moved into `tsconfig.json` instead?

Reply:
We could do that. I don't have a strong preference.